File: 1.data_structures_and_algorithms/3.bohubrihi_dsa_sakshar_chakravarty/1.introduction_module_1/1_sum_of_two_digits/APlusB.py
# python3
def sum_of_two_digits(first_digit, second_digit):
    return first_digit + second_digit

if __name__ == '__main__':
    # a, b = map(int, input().split())
    a = int(input("Enter a integer number: "))
    b = int(input("Enter another integer number: "))
    c = sum_of_two_digits(a, b)
    print(f"Sum: {c}")

File: 2-data-structures-and-algorithms/3.bohubrihi_dsa_sakshar_chakravarty/1.introduction_module_1/1_sum_of_two_digits/APlusB.cpp
#include <iostream>

using namespace std;

int sum_of_two_digits(int first_digit, int second_digit) {
    return first_digit + second_digit;
}

int main() {
    int a = 0;
    int b = 0;
    cout << "Enter a integer number: ";
    cin >> a;
    cout << "Enter another integer number: ";
    cin >> b;
    cout << "Sum: " << sum_of_two_digits(a, b);
    return 0;
}
